src/components/MerchantsPage/MerchantsPage.tsx

```diff
@@ -76,12 +76,14 @@ const MerchantsPage = (props: Props) => {
           store!.locations.length > 0 && store!.locations[0].city === type
       );
       if (type === 'Floral Park') {
-        const flushing = sellers.filter(
+        const queens = sellers.filter(
```

Comment:
This logic confuses me, why is if (floral park) the top level, and does that need to be in the or statement? In any case I trust that this works 😬 thanks for hopping on this!

Reply:
so it seems like floral park is the default as it was the first so it was always classified as "queens"

Reply:
Ahhh makes sense, thanks for the clarification!